jumeg/connectivity/con_viz.py

```diff
@@ -700,22 +708,51 @@ def _get_node_grouping(label_groups, orig_labels):
     return node_order, group_numbers
 
 
-def _get_group_node_angles_and_colors(label_groups, orig_labels, cortex_colors=None):
-    if cortex_colors is None:
-        cortex_colors = ['m', 'b', 'y', 'c', 'r', 'g',
-                         'g', 'r', 'c', 'y', 'b', 'm']
+def _get_group_node_angles(label_groups, orig_labels):
```

Comment:
Would be great if you could include docstrings to the new functions esp. describing label_groups and orig_labels.